addons/stock/models/stock_quant.py

```diff
@@ -89,6 +123,30 @@ def check_location_id(self):
     def _compute_name(self):
         self.name = '%s: %s%s' % (self.lot_id.name or self.product_id.code or '', self.quantity, self.product_id.uom_id.name)
 
+    def _get_inventory_move_values(self, location_id, location_dest_id, quantity):
```

Comment:
missing docstring, + it's really similar to _get_move_values of inventory adj, it's worth having a look if we could share the code between them